src/qmapshack/mouse/line/IMouseEditLine.cpp

```diff
@@ -96,7 +97,7 @@ void IMouseEditLine::commonSetup() {
   connect(scrOptEditLine->toolDeletePoint, &QPushButton::clicked, this, &IMouseEditLine::slotDeletePoint);
 
   connect(scrOptEditLine->toolNoRoute, &QPushButton::clicked, this, &IMouseEditLine::slotNoRouting);
-  connect(scrOptEditLine->toolAutoRoute, &QPushButton::clicked, this, &IMouseEditLine::slotAutoRouting);
+//connect(scrOptEditLine->toolAutoRoute, ...) now handled by function slotHasFastRouting
   connect(scrOptEditLine->toolVectorRoute, &QPushButton::clicked, this, &IMouseEditLine::slotVectorRouting);
```

Comment:
if it is not used, remove it

Reply:
Statement "connect(scrOptEditLine->toolAutoRoute, ..." moves back from "IMouseEditLine::slotTrackRouting" to constructor.